contracts/service-registry/src/msg.rs

```diff
@@ -66,6 +71,13 @@ pub enum ExecuteMsg {
     ClaimStake { service_name: String },
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct VerifierDetailsResponse {
```

Comment:
```suggestion
pub struct VerifierDetails {
```
up to you, but I feel like this is cleaner. The `Response` suffix always feels so ugly to me

Reply:
yeah makes sense. Updated!